tests/Cms/App/fixtures/controllers/another.json.php

```diff
@@ -1,3 +1,5 @@
 <?php
 
-echo json_encode(['foo' => 'bar']);
```

Comment:
`Question:` Why would the controller echo the JSON? Isn't that the job of the template in a content representation?

Reply:
I have no idea. The controller does not make any sense in that form.